mxd/modules/connector/azurite-container.tf

```diff
@@ -0,0 +1,75 @@
+#
+#  Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+resource "kubernetes_job" "azurite-container" {
```

Comment:
not for this PR, but we should keep seeding the data backends (Azurite) and the EDC data (via management API) consistent, meaning:
either we create one script to seed both connectors (`seed_data.tf`), or we create the seed script "per connector", like you did with Azurite. There is advantages to either variant.